src/Components/Comments.jsx

```diff
@@ -16,6 +19,13 @@ const Comments = ({ article_id }) => {
     setComments([...comments, newComment]);
   };
 
+  const deleteComment = (deletedId) => {
+    const newComments = comments.filter(
+      (comment) => comment.comment_id !== deletedId
+    );
+    setComments(newComments);
+  };
+
   const reversedComments = [...comments].reverse();
 
```

Comment:
I'd try and avoid sorting data in the front end, might be better to have this dealt with in the back end... maybe make a note to come back to this

Reply:
thanks for the heads up! i have sorted out how a user who is not signed in can write a comment (by signing in) sorted out posting offline by catching an error and returning an error message both when adding and deleting a comment